bundles/org.openhab.binding.shelly/src/main/resources/ESH-INF/thing/channels.xml

```diff
@@ -365,4 +353,20 @@
 			</options>
 		</event>
 	</channel-type>
+
+	<channel-type id="timeoutErrors" advanced="true">
```

Comment:
No, please don't add such debugging channels to the Things, this really feels not right.
Doing debug level logging should normally be enough. If you really want it somewhere else, maybe it would be ok to add it as Thing properties.

Reply:
ok, was designed as a temp at the beginning, but provided some good feedback
nevertheless I'll move it to thing properties as stats values. I'm currently working with the community and the Shelly guys on REST timeouts. Various users are reporting issues whereas others (like me) see timeouts very rarely. Once the problem is fixed (maybe a firmware issue) I Could remove those counters anyways.

Reply:
moved to thing properties

Reply:
still here - more commits missing?

Reply:
left over, removed